src/Functions/DateTimeTransforms.h

```diff
@@ -61,15 +61,22 @@ struct ToDateImpl
 
     static inline UInt16 execute(Int64 t, const DateLUTImpl & time_zone)
     {
-        return UInt16(time_zone.toDayNum(t));
+        if (t < 0)
```

Comment:
Just wondering: does the input `Int64` (l. 62) represent the ClickHouse data type "Int64" or the ClickHouse data type "DateTime64"? In case of the latter, I wonder what a negative DateTime64 means.

Reply:
A negative DT64 means timepoint prior to 1970